packages/api/tests/endpoints/test-stats.js

```diff
@@ -100,15 +104,18 @@ test.before(async (t) => {
     }))
   ));
 
-  await t.context.collectionPgModel.insert(
-    t.context.knex,
-    collections
-  );
+  range(24).map((num) => (
```

Comment:
nit: I think this should be foreach since we're not returning anything from it

Reply:
hmm... I did it this way because so many of the other tests are doing it the same way

Reply:
I think if I make this change, then to be consistent I need to make a similar change to the other data types here (collections, granules) and in most of these other test files (endpoint tests, search tests). Is that what you think I should do?

Reply:
You can just update the files you added/updated.  The original implementation is not wrong, just not as simple.

Reply:
I can do that, but I feel like we should have some way of noting that if anyone makes changes to the other files, they should make a similar change...